sdk/src/main/java/io/dapr/actors/DaprAsyncClient.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ */
+
+package io.dapr.actors;
+
+import reactor.core.publisher.Mono;
+
+/**
+ * Interface for interacting with Dapr runtime.
+ */
+public interface DaprAsyncClient {
+
+    /**
+     * Invokes an Actor method on Dapr.
+     * @param actorType Type of actor.
+     * @param actorId Actor Identifier.
+     * @param methodName Method name to invoke.
+     * @param jsonPayload Serialized body.
+     * @return Asynchronous result with the Actor's response.
+     */
+    Mono<String> invokeActorMethod(String actorType, String actorId, String methodName, String jsonPayload);
+
+    /**
+     * Gets a state from Dapr's Actor.
+     * @param actorType Type of actor.
+     * @param actorId Actor Identifier.
+     * @param keyName State name.
+     * @return Asynchronous result with current state value.
+     */
+    Mono<String> getState(String actorType, String actorId, String keyName);
+
+    /**
+     * Saves Actor state in Dapr. This is temporary until the Dapr runtime implements the Batch state update.
+     * @param actorType Type of actor.
+     * @param actorId Actor Identifier.
+     * @param keyName State name.
+     * @param data State to be saved.
+     * @return Asynchronous void result.
+     */
+    Mono<Void> saveState(String actorType, String actorId, String keyName, String data);

```

Comment:
We can remove this saveState, it will not be used as Actors will always save transactionally, Thsi will be rmeoved from dotnet-sdk as well.

Reply:
Done.